src/Tribe/Repository/Decorator.php

```diff
@@ -486,7 +486,7 @@ public function filter_postarr_for_create( array $postarr ) {
 	 * {@inheritdoc}
 	 */
 	public function set_create_args( array $create_args ) {
-		$this->decorated->set_create_args( $create_args );
+		return $this->decorated->set_create_args( $create_args );
 	}
```

Comment:
According to the inherited doc there are no returns, should we update the Docblock in the parent? Or just update within decorator?

Reply:
I'll update the interface docs, good eye thanks!